src/components/CreateNewProjectWizard/Gatsby/ProjectStarterSelection.test.js

```diff
@@ -84,12 +84,12 @@ describe('ProjectStarterSelection component', () => {
   it('should deselect starter if selectedStarter clicked', () => {
     wrapper.setProps({ projectStarter: 'blog' });
     instance.handleOnSelect('blog', true);
-    expect(mockedOnSelect).toBeCalledWith('');
+    expect(mockedOnSelect).toHaveBeenCalledWith('');
```

Comment:
Curious what's the difference between `toHaveBeenCalledWith` and `toBeCalledWith`?